components/person/PersonMetaTags.vue

```diff
@@ -5,14 +5,17 @@
     props: {
       person: {
         type: Object,
-        required: true,
+        required: true
       }
     },
 
     head () {
-      const personPhotos = this.person.photos && this.person.photos.length ?  this.person.photos[0].url : ''
+      const personPhotos = this.person.photos && this.person.photos.length
+        ? this.person.photos[0].url : ''
+
       return {
-        title: this.person.name,
+        title: `Ayudanos a encontrar a ${this.person.name}`,
```

Comment:
Is not bad a long title, is something normal. Is better for SEO and is more expressive. Using just the name seems to be a profile page, something like this is more meaningful 

<img width="371" alt="screen shot 2017-04-20 at 4 42 44 am" src="https://cloud.githubusercontent.com/assets/3258966/25219169/da76b314-2583-11e7-86c0-563b42516f1e.png">